content/quickstart/kubernetes/tutorial-azure-kubernetes-service.md

```diff
@@ -1,9 +1,13 @@
 ---
-title: "Tutorial: Azure Kubernetes Service"
-aliases: ["tutorial-azure-kubernetes-service.html"]
+title: "Tutorial: AKS - Hello World!"
+aliases:
+    - tutorial-aks.html
+    - /quickstart/kubernetes/tutorial-azure-kubernetes-service
```

Comment:
```suggestion
    - /quickstart/azure/tutorial-azure-kubernetes-service.html
    - /quickstart/azure/tutorial-azure-kubernetes-service/
```

There are two aliases (redirects) needed for this file:

1. One for the old pre-Hugo ugly style URL: `/quickstart/azure/tutorial-azure-kubernetes-service.html`
2. One for the Hugo pretty style URL at the previous location: `/quickstart/azure/tutorial-azure-kubernetes-service/`